docs/articles/nunit/release-notes/framework.md

```diff
@@ -2,9 +2,151 @@
 <!-- markdownlint-disable-file MD013 -->
+<!-- markdownlint-disable-file MD033 -->
 
 # Framework Release
 
+## NUnit 4.3.0 -  ETA Dec 15. 2024
+
+This release includes a series of enhancements and bug fixes. Notably, some of the bug fixes address issues related to a lack of type safety in NUnit. See issues [4877](https://github.com/nunit/nunit/issues/4877),
+[4876](https://github.com/nunit/nunit/issues/4876), [4875](https://github.com/nunit/nunit/issues/4875), [4874](https://github.com/nunit/nunit/issues/4874) and [150](https://github.com/nunit/nunit/issues/150).  
+
+Be aware that this might cause compile errors in cases where the code was previously ignored. These errors can be safely resolved by removing the affected code..
+
+There are 27 issues fixed in this release.
+
+### Enhancements
+
+* [4890](https://github.com/nunit/nunit/issues/4890) Add .net 8 as a target framework in the package. Thanks to NUnit Team member [Manfred Brands](https://github.com/manfred-brands) for [PR 4884]( https://github.com/nunit/nunit/pull/4884)
+* [4886](https://github.com/nunit/nunit/issues/4886) Optimize ToArray()[0] with First() in TestAssert class in TestAssert.cs. Thanks to [](https://github.com/Saurav-shres) for [PR 4888](https://github.com/nunit/nunit/pull/4888)
+* [4862](https://github.com/nunit/nunit/issues/4862) PropertiesComparer 32 Property Limit Error Message. Thanks to [Cincidial](https://github.com/Cincidial) for [PR 4870](https://github.com/nunit/nunit/pull/4870)
+* [4861](https://github.com/nunit/nunit/issues/4861) Always repeat Test x times. Thanks to [Christoph](https://github.com/cfuerbachersparks) for [PR 4864](https://github.com/nunit/nunit/pull/4864)
+* [4852](https://github.com/nunit/nunit/issues/4852) Expand the usability of `Contains.Key(...).WithValue(...)`. Thanks to [Michael Render](https://github.com/RenderMichael) for [PR 4854](https://github.com/nunit/nunit/pull/4854)
+* [4846](https://github.com/nunit/nunit/issues/4846) Add NET9 target for the framework tests. Thanks to [Leon Lux](https://github.com/Lachstec) for [PR 4851](https://github.com/nunit/nunit/pull/4851)
+* [4839](https://github.com/nunit/nunit/issues/4839) Allow to use collection expressions with collection constraints. Thanks to [Oleksandr Liakhevych](https://github.com/Dreamescaper) for [PR 4840](https://github.com/nunit/nunit/pull/4840)
+* [4832](https://github.com/nunit/nunit/issues/4832) Setting thread names to make debugging easier. Thanks to NUnit Team member [Terje Sandstrom](https://github.com/OsirisTerje) for [PR 4835]( https://github.com/nunit/nunit/pull/4835)
+* [4811](https://github.com/nunit/nunit/issues/4811) Assert.ThatAsync does not support polling. Thanks to [Oleksandr Liakhevych](https://github.com/Dreamescaper) for [PR 4813](https://github.com/nunit/nunit/pull/4813)
+* [4771](https://github.com/nunit/nunit/issues/4771) Support tuples in inline data. Thanks to [](https://github.com/MaxKot) for [PR 4772](https://github.com/nunit/nunit/pull/4772)
```

Comment:
`Thanks to []` will not show anything in the resulting markdown

Reply:
Fixed